pkgs/development/python-modules/py-slvs/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, blender, python3Packages, pkgs, fetchurl }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "py-slvs";
+  version = "1.0.6";
+
+  src = python3Packages.fetchPypi {
+    pname = "py_slvs";
+    version = "1.0.6";
+    sha256 = "sha256-U6T/aXy0JTC1ptL5oBmch0ytSPmIkRA8XOi31NpArnI=";
+  };
+
+  pyproject = true;
+
+  nativeBuildInputs = with pkgs; [
+    swig
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
```

I'd recommend changing over from `propagatedBuildInputs` to the new naming for build-time python dependencies: `build-system`.

If you're wondering: this naming is taken directly from `pyproject.toml`, which also uses `build-system`

```suggestion
  build-system = [
```